### PR TITLE
Move torsoriality of the identity type to `foundation-core.torsorial-type-families`

### PR DESCRIPTION
**Pull request by EgbertRijke**
This PR moves the torsoriality of the identity types to `foundation-core.torsorial-type-families`. Previously it was in `contractible-types`. I also slightly updated the prose, and fixed imports wherever they were broken because of this move.

### CODE REVIEW

File: src/foundation/families-of-maps.lagda.md [hunk 1]
```diff
@@ -52,17 +53,21 @@ module _
...
-          Σ (Σ A (x ＝_)) λ (x' , _) → C x') by equiv-Π-equiv-family (λ (x , _) →
-                                                  inv-left-unit-law-Σ-is-contr
-                                                    (is-torsorial-Id x)
-                                                    (x , refl))
-      ≃ (((x , _) : Σ A B) →
-          Σ (Σ A C) λ (x' , _) → x ＝ x') by equiv-Π-equiv-family (λ (x , _) →
-                                              equiv-right-swap-Σ)
-      ≃ Σ (Σ A B → Σ A C)
-          (λ f → pr1 ~ (pr1 ∘ f)) by distributive-Π-Σ
+          Σ (Σ A C) λ (x' , _) → x ＝ x')
```
Comment (fredrik-bakke) [inline src/foundation/families-of-maps.lagda.md:66]:
```suggestion
          Σ (Σ A C) (λ (x' , _) → x ＝ x'))
```

File: src/foundation/families-of-maps.lagda.md [hunk 2]
```diff
@@ -52,17 +53,21 @@ module _
   equiv-fam-map-map-tot-space =
     equivalence-reasoning
       fam-map
-      ≃ (((x , _) : Σ A B) → C x) by equiv-ind-Σ
+      ≃ (((x , _) : Σ A B) → C x)
+        by equiv-ind-Σ
+      ≃ (((x , _) : Σ A B) → Σ (Σ A (x ＝_)) λ (x' , _) → C x')
```
Comment (fredrik-bakke) [inline src/foundation/families-of-maps.lagda.md:58]:
```suggestion
      ≃ (((x , _) : Σ A B) → Σ (Σ A (x ＝_)) (λ (x' , _) → C x'))
```

File: src/foundation/families-of-maps.lagda.md [hunk 3]
```diff
@@ -52,17 +53,21 @@ module _
...
+        by
+        equiv-Π-equiv-family
+          ( λ (x , _) →
+              inv-left-unit-law-Σ-is-contr
+                ( is-torsorial-Id x)
+                ( x , refl))
```
Comment (fredrik-bakke) [inline src/foundation/families-of-maps.lagda.md:64]:
```suggestion
            inv-left-unit-law-Σ-is-contr
              ( is-torsorial-Id x)
              ( x , refl))
```

File: src/foundation/families-of-maps.lagda.md [hunk 4]
```diff
@@ -52,17 +53,21 @@ module _
...
-                                                    (x , refl))
-      ≃ (((x , _) : Σ A B) →
-          Σ (Σ A C) λ (x' , _) → x ＝ x') by equiv-Π-equiv-family (λ (x , _) →
-                                              equiv-right-swap-Σ)
-      ≃ Σ (Σ A B → Σ A C)
-          (λ f → pr1 ~ (pr1 ∘ f)) by distributive-Π-Σ
+          Σ (Σ A C) λ (x' , _) → x ＝ x')
+        by
+        equiv-Π-equiv-family (λ (x , _) → equiv-right-swap-Σ)
+      ≃ Σ (Σ A B → Σ A C) (λ f → pr1 ~ (pr1 ∘ f))
```
Comment (fredrik-bakke) [inline src/foundation/families-of-maps.lagda.md:69]:
```suggestion
      ≃ Σ (Σ A B → Σ A C) (λ f → pr1 ~ pr1 ∘ f)
```

File: src/foundation/families-of-maps.lagda.md [hunk 5]
```diff
@@ -52,17 +53,21 @@ module _
...
-                                                    (is-torsorial-Id x)
-                                                    (x , refl))
-      ≃ (((x , _) : Σ A B) →
-          Σ (Σ A C) λ (x' , _) → x ＝ x') by equiv-Π-equiv-family (λ (x , _) →
-                                              equiv-right-swap-Σ)
-      ≃ Σ (Σ A B → Σ A C)
-          (λ f → pr1 ~ (pr1 ∘ f)) by distributive-Π-Σ
+          Σ (Σ A C) λ (x' , _) → x ＝ x')
+        by
+        equiv-Π-equiv-family (λ (x , _) → equiv-right-swap-Σ)
```
Comment (fredrik-bakke) [inline src/foundation/families-of-maps.lagda.md:68]: Not sure if this pattern matching was used to demonstrate something?

```suggestion
        equiv-Π-equiv-family (λ _ → equiv-right-swap-Σ)
```

**Review by fredrik-bakke — APPROVED** [review]
Super!

**Comment by fredrik-bakke** [comment]
Would it be nice to also mention in the `## Idea` section that the identity type family is the motivating example of the definition?

**Comment by VojtechStep** [comment]
I'd expect this property to be recorded in `foundation.identity-types` - are there circular dependency issues with that?

**Comment by fredrik-bakke** [comment]
> > Question: When running `make website` I get the following error. What should I do to make it go away?
> > ```
> > Traceback (most recent call last):
> >   File "/Users/egbertrijke/Repositories/agda-unimath/./scripts/preprocessor_citations.py", line 8, in <module>
> >     import pybtex
> > ModuleNotFoundError: No module named 'pybtex'
> > ```
> 
> You have to install the latest requirements. What you're missing is the `pybtex` module

The command would be `pip3 install -r scripts/requirements.txt` unless you've recently updated python with brew. If so, the command is `pip3 install -r scripts/requirements.txt --break-system-packages`

File: src/foundation/families-of-maps.lagda.md [hunk 6]
```diff
@@ -73,16 +78,17 @@ module _
...
+          ( λ (e , _) → is-equiv e)
+        by
+        equiv-subtype-equiv
+          ( equiv-fam-map-map-tot-space)
+          ( λ f → Π-Prop A (is-equiv-Prop ∘ f))
+          ( λ (e , _) → is-equiv-Prop e)
+          ( λ f →
+            is-equiv-tot-is-fiberwise-equiv , is-fiberwise-equiv-is-equiv-tot)
+      ≃ Σ (Σ A B ≃ Σ A C) (λ e → pr1 ~ (pr1 ∘ map-equiv e))
+        by equiv-right-swap-Σ
```
[the remark this answers is not in the document]
Comment (EgbertRijke) [reply src/foundation/families-of-maps.lagda.md]: There were even more.

I could almost change the title of this PR to "Formatting edits for families-of-maps" 😅

**Comment by EgbertRijke** [comment]
> I'd expect this property to be recorded in `foundation.identity-types` - are there circular dependency issues with that?

You're right, and yes that leads to cyclic module dependencies that I don't want to solve now.

**Comment by EgbertRijke** [comment]
> Would it be nice to also mention in the `## Idea` section that the identity type family is the motivating example of the definition?

On it.

**Comment by EgbertRijke** [comment]
I removed "chore" from the title of this PR because I wrote a somewhat substantial amount of text and that should definitely count as content creation.

**Pull request merged**